snakemake/script.py

```diff
@@ -563,5 +564,7 @@ def script(
     except URLError as e:
         raise WorkflowError(e)
     finally:
-        if f:
+        if f and cleanup_scripts:
             os.remove(f.name)
+        else:
+            logger.debug("Not cleaning up %s" % f.name)
```

Comment:
Maybe even have a regular logger.warning here. This way, people are not suprised that the file is still there and pollutes their workdir.

Reply:
All set.